develop/tutorials/articles/110-portlets/02-liferay-soy-portlet/01-creating-a-soy-portlet.markdown

```diff
@@ -74,7 +74,7 @@ you should have the following dependencies and configuration parameters:
         }
 
 This provides everything you need to create a Metal component based on Soy. Note 
-that the values of `name` and `version` should match those in your `bnd.bnd` 
+that the values of `name` and `version` should match the values of `Bundle-Name` and `Bundle-Version` in your `bnd.bnd` 
```

Comment:
name doesn't really need to match anything, but should it, it would be the Web-ContextPath property